### ISSUE TITLE
Increasing the resolution scale more than 2x does not improve the image quality.

### ISSUE THREAD

**Issue by Ulexer** [UltimateBoomer/Resolution-Control#47]
Linear interpolation is not suitable for scales larger than 2x. With linear interpolation, a resolution scale above 2x makes no sense other than performance testing. Instead, it is better to use averaging. [Here is](https://imgur.com/0hLvuI2) an example showing how bad linear interpolation is.

**Comment by christopher317** [comment]
agreed, i would say it looks fine for 1x through 2x ( if even slightly better ) but for 3x+ it makes it far worse than 2x.
As Ulexer said, it would be better to use averaging 

1x:
![image](https://github.com/UltimateBoomer/Resolution-Control/assets/86526198/79988c1d-88d9-4ab5-9bb2-eac0602fb1d6)

2x:
![image](https://github.com/UltimateBoomer/Resolution-Control/assets/86526198/6e150b9c-5a89-4f54-b092-d6f4229b1f9b)

4x:
![image](https://github.com/UltimateBoomer/Resolution-Control/assets/86526198/3dc90ed9-81e0-40b0-8009-9aa3e9727ff7)

8x:
![image](https://github.com/UltimateBoomer/Resolution-Control/assets/86526198/382952ab-dda0-44b3-947f-5914bee941b8)
